net/Socket.cpp

```diff
@@ -555,6 +555,33 @@ void StreamSocket::dumpState(std::ostream& os)
         Util::dumpHex(os, "\t\toutBuffer:\n", "\t\t", _outBuffer);
 }
 
+void StreamSocket::sendError(int errorCode,
+               const std::string& body, const std::string& extraHeader, bool shutdownAfter)
+{
+
+    std::ostringstream oss;
+    oss << "HTTP/1.1 " << errorCode << "\r\n"
+        << "Date: " << Util::getHttpTimeNow() << "\r\n"
+        << "User-Agent: " << WOPI_AGENT_STRING << "\r\n"
+        << extraHeader;
+    if (body.empty())
+    {
+        oss << "Content-Length: 0\r\n";
+    }
+    oss << "\r\n";
+
+    if (!body.empty())
```

Comment:
Why not just unconditionally output the body? If it's empty, nothing is amended to oss.

`oss << body;`

Reply:
:+1:  d398545e5